src/widgets/table/table.rs

```diff
@@ -237,7 +237,7 @@ impl<'a> Default for Table<'a> {
             highlight_style: Default::default(),
             highlight_symbol: Default::default(),
             highlight_spacing: Default::default(),
-            segment_size: SegmentSize::None,
+            flex: Flex::Start,
```

Comment:
I personally think the default should be `Flex::Stretch` for now. We should change it along with the default for `Layout` to be `Flex::Start` in a future PR.

Reply:
But isn't `Flex::Start` the equivalent of `SegmentSize::None`? I think the opposite should be done.

Reply:
`Flex::Start` is the equivalent of `SegmentSize::None` but imho that's not a good default. 

Let's see what others think.

Reply:
I think we need to split the task of deprecating the existing behavior and changing the defaults into two separate tasks. Combining the two is problematic. The first is a mechanical change that is either right or wrong, the second is an exercise in opinion, and relies on specifying the existing behavior and understanding the impact of changing it.

Reply:
Fair point. I'll resolve this conversation.